openwisp_users/tests/test_admin.py

```diff
@@ -493,6 +493,44 @@ def test_organization_uuid_field(self):
         html = '<input type="text" name="name" value="default"'
         self.assertNotContains(response, html)
 
+    def test__action_active(self):
+        user = User.objects.create(is_active=False)
+        path = reverse('admin:openwisp_users_user_changelist')
+        self.client.force_login(self._get_admin())
+        post_data = {
```

Comment:
indentation looks weird, please make sure it's like:
```python
post_data = {
    'key': 'value'
}
```
same applies for the other test methods